src/core/public/core_app/status/lib/load_status.ts

```diff
@@ -57,7 +57,7 @@ function formatMetrics({ metrics }: StatusResponse): Metric[] {
         defaultMessage: 'Load',
       }),
       value: [metrics.os.load['1m'], metrics.os.load['5m'], metrics.os.load['15m']],
-      type: 'time',
+      type: 'float',
```

Comment:
@joshdover The `time` type was being [formatted](https://github.com/joshdover/kibana/blob/status-load-label/src/core/public/core_app/status/lib/format_number.ts#L20-L23) with a special case. I'm not sure if that's not being read properly or got lost somewhere along the line.

Reply:
Yep that's the underlying reason this was happening. Probably something I broke a while back to be honest. I didn't remove it since we're still using this for the `response_times` metrics.